packages/lab/src/pill-next/PillNext.css

```diff
@@ -0,0 +1,86 @@
+/* Styles applied to the root element */
+.saltPillNext {
+  --pillNext-background: var(--saltPillNext-background, var(--salt-actionable-primary-background));
+  --pillNext-background-active: var(--saltPillNext-background-active, var(--salt-actionable-primary-background-active));
+  --pillNext-background-disabled: var(--saltPillNext-background-disabled, var(--salt-actionable-primary-background-disabled));
+  --pillNext-background-hover: var(--saltPillNext-background-hover, var(--salt-actionable-primary-background-hover));
+  --pillNext-text-color: var(--saltPillNext-text-color, var(--salt-actionable-primary-foreground));
+  --pillNext-text-color-active: var(--saltPillNext-text-color-active, var(--salt-actionable-primary-foreground-active));
+  --pillNext-text-color-hover: var(--saltPillNext-text-color-hover, var(--salt-actionable-primary-foreground-hover));
+  --pillNext-text-color-disabled: var(--saltPillNext-text-color-disabled, var(--salt-actionable-primary-foreground-disabled));
+}
+
+.saltPillNext {
+  display: inline-flex;
+  align-items: center;
+  background: var(--pillNext-background);
+  border-radius: var(--saltPillNext-borderRadius, 0);
+  height: calc(var(--salt-size-base) - var(--salt-spacing-100));
+  /* max-width: var(--saltPillNext-maxWidth, 10em); */
+  min-width: var(--saltPillNext-minWidth, 4em);
+  min-height: var(--saltPillNext-minHeight, var(--salt-text-minHeight));
+  outline: var(--saltPillNext-outline, 0);
+  position: relative;
+  gap: var(--saltPillNext-gap, var(--salt-spacing-50));
+  padding-inline: var(--salt-spacing-50);
+  color: var(--saltPillNext-text-color, var(--pillNext-text-color));
+  font-size: var(--saltPillNext-fontSize, var(--salt-text-fontSize));
+  font-weight: var(--saltPillNext-fontWeight, var(--salt-text-fontWeight));
+  line-height: var(--saltPillNext-lineHeight, var(--salt-text-label-lineHeight));
+}
+
+.saltPillNext-label {
+  min-width: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.saltPillNext .saltIcon {
+  --saltIcon-color: currentColor;
+}
+
+/* Style applied to Pill if `onClick` prop is provided  */
+.saltPillNext-clickable {
+  cursor: var(--salt-selectable-cursor-hover);
+  user-select: none;
+}
+
+.saltPillNext-clickable:hover {
+  color: var(--pillNext-text-color-hover);
+  background: var(--pillNext-background-hover);
+}
+
+.saltPillNext-clickable.saltPillNext-disabled:hover,
+.saltPillNext-nestedHover:hover {
```

Comment:
Is it worth removing the nestedHover classes as they are not needed for V1? We can always look at the commit it we need them again.

Reply:
Yep, true. Will remove